examples/tests/sdk_tests/bucket/rename.test.w

```diff
@@ -0,0 +1,67 @@
+bring cloud;
+bring util;
+bring expect;
+
+let b = new cloud.Bucket();
+
+test "rename()" {
+  let assertThrows = (expected: str, block: (): void) => {
+    let var error = false;
+    try {
+      block();
+    } catch actual {
+      assert(actual.contains(expected));
```

Comment:
Do you think it will be ok to change to `actual == expected`? to make sure all the error messages are aligned in this test for all targets? (As I found one manually)